python/hyperspace/hyperspace.py

```diff
@@ -0,0 +1,173 @@
+import sys
+
+from pyspark.sql import SparkSession, DataFrame
+from hyperspace import *
+from py4j.java_gateway import java_import
+
+
+class Hyperspace:
+    def __init__(self, spark):
+        """
+        Initializes hyperspace object.
+        :param spark: sparkSession object
+        :return: hyperspace object
+
+        >>> hyperspace = Hyperspace(spark)
+        """
+        self.sparkContext = spark.sparkContext
+        self.spark = spark
+        self.jvm = spark._jvm
+        self.hyperspace = self.jvm.com.microsoft.hyperspace.Hyperspace(spark._jsparkSession)
+
+    def __getJavaIndexConfig(self, index_config):
+        """
+        Constructs IndexConfig Java object from python wrapper IndexConfig object.
+        :param index_config: IndexConfig java object
+        :return: IndexConfig python object
+
+        >>> __getJavaIndexConfig(idx_config)
+        """
+        indexed_columns = self.__getScalaSeqFromList(index_config.indexedColumns)
+        included_columns = self.__getScalaSeqFromList(index_config.includedColumns)
+        _jindexConfig = self.jvm.com.microsoft.hyperspace.index.IndexConfig(
+                     self.jvm.String(index_config.indexName), indexed_columns, included_columns)
+        return _jindexConfig
+
+    def __getScalaSeqFromList(self, list):
```

Comment:
PEP-8 says these non-public methods names should have a single leading `_`:  https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables

I'm also wondering if it would be better to make these helper methods static and put them in some Util class.

Reply:
Thanks. I updated to _ prefix.

Reply:
For static Utils question, if we follow similar style as spark or delta, they have minimal number of py packages and they have put private functions used inside package in same file. Ex. https://github.com/delta-io/delta/blob/master/python/delta/tables.py